### DISCUSSION TITLE
Studio README questions

### DISCUSSION THREAD

**Discussion by charles-hill-work** [apollographql/apollo-studio-community#147] [Q&A] [answered]
1. What flavor of markdown is used?  Can we get a link to the doc's so that we know what syntax is available?
2. How do we embed Variables with our GQL, so that clicking Open in Explorer (play button) includes a fully working sample?

**Comment by cheapsteak** [comment, marked as answer]
1. We're currently using https://www.npmjs.com/package/markdown-to-jsx, but can't commit to not switching away from it later (there's a potential that we might switch to `remark`)
2. Good question 💭 do you have ideas on what that might look like? (specifically where might be a good place to provide the variables)

**Comment by cheapsteak** [comment]
oo we might have a different way of providing variables, hang tight for a bit 🙏  (~ a week)

**Reply by charles-hill-work** [reply]
Right now, we're including our sample variables as a json code block below the query.  My best idea so far is it could be included in the same block as the graphql, using a special comment syntax to differentiate the two sections
```graphql
query something($input: String!) {
  resource(input: $input): {
    name
  }
}
# Variables
{
  "input": "something"
}
````

**Reply by cheapsteak** [reply]
Would the newly released [operation collections](https://www.apollographql.com/docs/studio/explorer/operation-collections/#embedding-in-the-readme-page) solve for providing READMEs with variables?

**Reply by charles-hill-work** [reply]
I think so, yes!  Links for the win
